.github/workflows/flask.yml

```diff
@@ -0,0 +1,36 @@
+name: Flask
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
`Pip install` causes `ERROR: Could not find a version that satisfies the requirement mock==4.0.2`. we should be using `pip3`?

Reply:
If you have both Python 2 and 3 installed in your environment, you should use `pip3` instead. In some other environments that uses version management tools such as pyenv then `pip` will be enough.

For the github action workflow, it uses an image with python 3 only so `pip` is good.

I added a note at the beginning of the readme to clarify this confusion.

Reply:
thank you 👍